colossus-metrics/src/main/scala/colossus/metrics/MetricClock.scala

```diff
@@ -17,40 +17,50 @@ class MetricDatabase(systemId: MetricSystemId, namespace: MetricAddress, snapsho
 
   def blankMap(): MetricMap = if (collectSystemMetrics) systemMetrics.metrics else Map()
 
-  var latestTick = 0L
-  var build: MetricMap = blankMap() 
+  var build: MetricMap = blankMap()
   var metrics = systemMetrics.metrics
   val collectors = new JHashSet[ActorRef]()
+  var latestTick = 0L
 
-  //TODO: this name needs to be able to support multiple databases/metric systems.
-  val collectedGauge = new ConcreteGauge(Gauge(namespace / "metric_completion"), self)
+  val metricSafeDurationName = interval.toString().replaceAll(" ", "") //ie: 1second, 44milliseconds
+  val collectedGauge = new ConcreteGauge(Gauge(namespace / metricSafeDurationName / "metric_completion"), self)
```

Comment:
FYI my merge broke this line I think, ConcreteGauge doesn't take the actor as a param anymore.